RecFCCeeCalorimeter/src/components/CaloTowerToolFCCee.h

```diff
@@ -124,7 +125,6 @@ class CaloTowerToolFCCee : public GaudiTool, virtual public ITowerToolThetaModul
    *   @param[in] aSegmentation Segmentation of the calorimeter
    */
   void CellsIntoTowers(std::vector<std::vector<float>>& aTowers, const edm4hep::CalorimeterHitCollection* aCells,
-                       dd4hep::DDSegmentation::Segmentation* aSegmentation, SegmentationType aType,
```

Comment:
if you change the signature of this function, please update also the documentation just above (remove   @param[in] aSegmentation Segmentation of the calorimeter, also please add the bool param)

Reply:
Changed as suggested